public/views/adminRights.html

```diff
@@ -0,0 +1,27 @@
+<h2>Configuration des droits des usagers</h2>
+<h4>Filtres</h4>
+<div class="row">
+    <div class="col-md-3">
+        Nom: <input type="text" class="form-control" ng-model="nameFilter" ng-change="filter('cipFilter', 'nameFilter')">
+    </div>
+    <div class="col-md-1" ng-repeat="right in rights">
```

Comment:
I don't think there is still a use to keep this form, as the user list below already does it all... and it's nicer...